examples/colab/Bangla_article_classifier.ipynb

```diff
@@ -0,0 +1,1000 @@
+{
```

Comment:
<p>please add space after each ",", ".", and "=" and any other operator or punctuation everywhere in the file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>